frontend/src/components/features.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import img1 from './images/review.svg'
+import img2 from './images/list.svg'
+import img3 from './images/eating.svg'
+import './features-style.css'
+
+const Feature = props=> {
+    return(
+        <div className="card text-center">
+            <div className="overflow">
+                <img src={props.imgsrc} alt="process" className="card-img-top"></img>
+            </div>
+            <div className="card-body text-dark">
+                <h4 className="card-title">{props.title}</h4>
+                <p className="card-text text-secondary">
+                {props.desc}
+                </p>
+            </div>
+        </div>
+    );
+}
```

Comment:
I love what you did here! You noticed that there was a lot of duplicate code so you abstracted some of it out! One thing I would like changed though is to change this from a const to a function.